doc/api/buffer.md

```diff
@@ -869,6 +869,10 @@ The index operator `[index]` can be used to get and set the octet at position
 `index` in `buf`. The values refer to individual bytes, so the legal value
 range is between `0x00` and `0xFF` (hex) or `0` and `255` (decimal).
 
+Since this operator is inherited from `Uint8Array`, its behavior on
+out-of-bounds access is exactly the same - i.e. getting returns `undefined`,
+setting does nothing.
+
```

Comment:
Nikolai uses `i.e.` correctly here as far as I can tell: "id est", not "exempli gratia."